etn-ccis/blui-react-native-workflows
# Update jsdocs

Fixes 5309.



#### Changes proposed in this Pull Request:

- Update js docs to follow syntax -  @param {string} title - The title of the book.
- 
-



#### Screenshots / Screen Recording (if applicable)

- 
<img width="790" alt="Screenshot 2024-02-14 at 1 47 23 PM" src="https://github.com/etn-ccis/blui-react-native-workflows/assets/133877691/28b129fd-5f74-4c36-a12f-c8cbf385410f">
<img width="488" alt="Screenshot 2024-02-14 at 2 06 56 PM" src="https://github.com/etn-ccis/blui-react-native-workflows/assets/133877691/55b039cf-14af-4192-be13-cc72d646aff4">



#### To Test:

- Hover on the component tag and check props are visible. We can add a link to the jsdocs for detailed props descriptions in future.



#### Any specific feedback you are looking for?

-
